docs/help.txt

```diff
@@ -33,6 +33,6 @@ Features
   When you are satisfied with the number of imported tweets, select the
   *ENOUGH! STOP IMPORTING TWEETS* button.
 
-  .. NOTE::
+  .. TIP::
```

Comment:
Why was this one changed to `TIP`? To me, `NOTE` seems rather appropriate for it :)

Reply:
hmm! I think it's because I am using NOTE for things you should read and TIP for things that you don't have to read but you might be interested in. if that makes sense? :)